js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/__stories__/AutomaterializeMiddlePanel.stories.tsx

```diff
@@ -19,32 +25,41 @@ const path = ['test'];
 
 export const Empty = () => {
   return (
-    <MockedProvider
-      mocks={[Evaluations.Single(path), buildRunStatusOnlyQuery('abcdef12', RunStatus.STARTED)]}
-    >
+    <MockedProvider mocks={[Evaluations.Single(path)]}>
       <div style={{width: '800px'}}>
-        <AutomaterializeMiddlePanel
-          assetKey={{path}}
-          assetHasDefinedPartitions={false}
-          selectedEvaluationId={undefined}
-        />
+        <AutomaterializeMiddlePanel assetKey={{path}} selectedEvaluationId={undefined} />
       </div>
     </MockedProvider>
   );
 };
 
 export const WithoutPartitions = () => {
   return (
-    <MockedProvider
-      mocks={[
-        Evaluations.Single(path, `${TEST_EVALUATION_ID + 1}`),
-        buildRunStatusOnlyQuery('abcdef12', RunStatus.STARTED),
-      ]}
-    >
+    <MockedProvider mocks={[Evaluations.Single(path, `${TEST_EVALUATION_ID + 1}`)]}>
       <div style={{width: '800px'}}>
         <AutomaterializeMiddlePanel
           assetKey={{path}}
-          assetHasDefinedPartitions={false}
+          definition={
+            buildAssetNode({
+              partitionKeysByDimension: [
+                buildDimensionPartitionKeys({
+                  partitionKeys: ['testing', 'testing2'],
+                }),
+              ],
+            }) as any
```

Comment:
?

I know these are `stories.tsx` and `fixtures.tsx` but it makes me concerned that there are deeper type issues.

Reply:
same as above